requirements.txt

```diff
@@ -1,3 +1,3 @@
 pycryptodome>=3.4
 requests>=2.5,<3.0
-six~=1.10
+six>=1.10,<2.0
```

Comment:
May I ask, what does this have to do with `six`'s major version? I may be incorrect, but I don't see the difference this change makes according to https://www.python.org/dev/peps/pep-0440/#compatible-release. Please elucidate.

Reply:
@masojus the current code is pinning to any (generally latest) version of 1.10.x. This PR is allowing 1.11.x, 1.12.x, etc.

The problem here is that it's causing conflicts during dependency resolution, so if I have a package that needs six > 1.11, I can't install both this package and the other, even though this package shouldn't be affected by updating six's minor version (second part of the version number).